app/serializers/api/open_street_map_config_serializer.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+#
```

Comment:
this should be an empty line.

and if you have one more minute you can fix the other warning 
![image](https://user-images.githubusercontent.com/1640378/83184918-14507a80-a122-11ea-9af9-0fae8cdd224b.png)

by doing:
module Api
     class OpenStreetMapConfigSerializer < ActiveModel::Serializer
         ...
     end
end

Reply:
I think we can ignore the other 2 warnings :+1: 

Good point @filipefurtad0 :+1: